.github/workflows/php.yml

```diff
@@ -62,18 +62,19 @@ jobs:
             run: ./vendor/bin/phpstan analyse -c phpstan.neon.dist
 
   phpunit:
-    name: PHPUnit Tests
-    runs-on: ubuntu-latest
-    strategy:
-        matrix:
-            php: ['7.1', '7.2', '7.3', '7.4']
-        fail-fast: false
-    steps:
+      name: PHPUnit
+      runs-on: ubuntu-18.04
```

Comment:
Have you tested to use `ubuntu-latest` ?

Reply:
testing :)